### PR TITLE
SurfaceObserver: split resized_to() into window_resized_to() and content_resized_to()

### PR DESCRIPTION
**Pull request by wmww**
Alternative to #1060

### CODE REVIEW

File: src/server/symbols.map [hunk 1]
```diff
@@ -1,4 +1,4 @@
-MIR_SERVER_0.1.5 {
+MIR_SERVER_0.1.6 {
```
Comment (wmww) [inline src/server/symbols.map]: I'm not sure this is what I was supposed to do

Comment (AlanGriffiths) [reply src/server/symbols.map]: https://mir-server.io/doc/dso_versioning_guide.html

Comment (wmww) [reply src/server/symbols.map]: So... this is the correct way to change a symbol and break backwards compatibility?

Comment (AlanGriffiths) [reply src/server/symbols.map]: Part of it. The soname needs bumping too.

Comment (wmww) [reply src/server/symbols.map]: Oh, what does it mean to bump the soname? Doc just says when to do it.

Comment (AlanGriffiths) [reply src/server/symbols.map]: 1. bump MIRSERVER_ABI
2. tools/update_package_abis.sh

Maybe you can improve the docs?

Comment (wmww) [reply src/server/symbols.map]: When you say MIRSERVER_ABI, you mean change `MIR_SERVER_0.1.5` to `MIR_SERVER_0.1.6` in src/server/symbols.map, or is MIRSERVER_ABI a different thing?

Comment (AlanGriffiths) [reply src/server/symbols.map]: src/server/CMakeLists.txt:166

Comment (wmww) [reply src/server/symbols.map]: Ack, will get it passing the tests and then push version bump

Comment (RAOF) [reply src/server/symbols.map]: Hm. We would traditionally call this `MIR_SERVER_1.5.0`, as that's the version this would be released in.

Comment (AlanGriffiths) [reply src/server/symbols.map]: Let's be consistent with ourselves and use MIR_SERVER_1.6.0 as that's the version this would be released in.

**Comment by wmww** [comment]
Like that?

**Pull request merged**